client/src/partials/fiscal/fiscal.js

```diff
@@ -58,6 +59,10 @@ function FiscalController($state, fiscalService, ModalService, Notify) {
     });
   }
 
+  function back() {
+    $window.history.back();
```

Comment:
We should make sure that this works properly with `$state` - is the state re-triggered?